packages/llvm/llvm.3.9/files/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+make=$1
+libdir=$2
+
+# Create build dir
+mkdir build
+cd build
+
+# Configure
+cmake ..
+#    -DLLVM_BUILD_LLVM_DYLIB:BOOL=ON \
+#    -DLLVM_LINK_LLVM_DYLIB:BOOL=ON \
+#    ..
+
+# Build
+grep "^ocaml_llvm.*/fast:$" Makefile | tr ":\n" " " | xargs "$make"
```

Comment:
This really has nothing to do with the build system (LLVM should have an ocaml_all target maybe but this is not the fault of cmake or autoconf; and it won't help you with 3.9 anyway). However, make doesn't have an option to list all valid targets! This is madness.

I believe llvm_libs should only contain LLVM-3.9 in the case of BUILD_DYLIB/LINK_DYLIB. Is that not the case?